packages/mobile/src/store/storeContext.ts

```diff
@@ -45,5 +45,9 @@ export const storeContext: CommonStoreContext = {
   audioPlayer,
   cognito,
   trackDownload,
+  generalAdmission: Config.GENERAL_ADMISSION,
```

Comment:
These might be better in the `env` object that lives in storeContext

Reply:
yes! way better. didn't know this was a thing